Backpack/Card/Classes/BPKDividedCard.h

```diff
@@ -35,6 +35,9 @@ NS_SWIFT_NAME(DividedCard) IB_DESIGNABLE @interface BPKDividedCard: BPKCard
  */
 @property(nonatomic, assign) IBInspectable UILayoutConstraintAxis orientation;
 
+@property(nonatomic, assign, nullable) UIView *primarySubview;
```

Comment:
@k0nserv I decided to move these so that consumers can access them for testing / other purposes. WDYT?

Reply:
Sounds good 👍

Reply:
Should be mentioned in `UNRELEASED.md`

Reply:
Ah ok!